forte/data/extractor/base_extractor.py

```diff
@@ -231,16 +231,25 @@ def extract(self, pack: DataPack,
         """
         pass
 
-    def remove_from_pack(self, pack: DataPack, instance: Annotation):
-        r"""Functionality: Remove the existing feature of the instance
+    def pre_evaluation_action(self, pack: DataPack,
+                instance: Annotation):
+        r"""Remove the existing feature of the instance
```

Comment:
The base extractor should be more general, please put something like this at the begining: "This function is performed on the pack before the evaluation stage, allowing one to perform some actions to before the evaluation."

Note that deleting stuff is just an example.

Also, mentioning that "by default, this function will not do anything"

Reply:
Got it.